procrastinate/sql/migrations/delta_0.8.1_002_close_fetch_job_race_condition.sql

```diff
@@ -0,0 +1,46 @@
+DROP TABLE IF EXISTS procrastinate_job_locks;
```

Comment:
We'll be explicit in the changelog that with those migrations, the workers will need to be stopped when running the migration.
Of course, one can always write better migrations.